incubator/hnc/internal/reconcilers/object.go

```diff
@@ -660,13 +659,16 @@ func (r *ObjectReconciler) syncPropagation(ctx context.Context, log logr.Logger,
 func (r *ObjectReconciler) shouldPropagateSource(log logr.Logger, inst *unstructured.Unstructured, dst string) bool {
 	selector, err := selectors.GetSelector(inst)
 	if err != nil {
-		log.Error(err, "Invalid value")
+		log.Error(err, "Cannot propagate")
 	}
```

Comment:
Why not `return false` immediately?

Reply:
In the case there are multiple errors, I _think_ it will be helpful for the developers to see all the errors in the log, wdyt?

Reply:
updated according to our discussion offline